src/pages/workspace/WorkspaceProfileSharePage.tsx

```diff
@@ -32,9 +32,11 @@ function WorkspaceProfileSharePage({policy}: WithPolicyProps) {
 
     const policyName = policy?.name ?? '';
     const id = policy?.id ?? '';
+    const adminEmail = policy?.owner ?? '';
```

Comment:
This should be the email of the current admin viewing the page.
```suggestion
    const adminEmail = session.email;
```